### ISSUE TITLE
NullPointerException in self-check

### ISSUE THREAD

**Issue by wvengen** [microg/android_packages_apps_UnifiedNlp#60]
Whenever the self-check in 1.5.0 receives a location from a backend, it says: _An exception occured during self-check. Please report this issue_. OsmAnd~ does receive the location successfully.

```
W/SelfCheck( 5923): Failed during check UnifiedNlp status
W/SelfCheck( 5923): java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.os.Bundle.containsKey(java.lang.String)' on a null object reference
W/SelfCheck( 5923): 	at org.microg.tools.selfcheck.NlpStatusChecks.isProvidingLastLocation(NlpStatusChecks.java:79)
W/SelfCheck( 5923): 	at org.microg.tools.selfcheck.NlpStatusChecks.doChecks(NlpStatusChecks.java:50)
W/SelfCheck( 5923): 	at org.microg.tools.ui.AbstractSelfCheckFragment.onCreateView(AbstractSelfCheckFragment.java:60)
W/SelfCheck( 5923): 	at android.support.v4.app.Fragment.performCreateView(Fragment.java:1962)
W/SelfCheck( 5923): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
W/SelfCheck( 5923): 	at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1248)
W/SelfCheck( 5923): 	at android.support.v4.app.BackStackRecord.run(BackStackRecord.java:738)
W/SelfCheck( 5923): 	at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:1613)
W/SelfCheck( 5923): 	at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:517)
W/SelfCheck( 5923): 	at android.os.Handler.handleCallback(Handler.java:739)
W/SelfCheck( 5923): 	at android.os.Handler.dispatchMessage(Handler.java:95)
W/SelfCheck( 5923): 	at android.os.Looper.loop(Looper.java:135)
W/SelfCheck( 5923): 	at android.app.ActivityThread.main(ActivityThread.java:5254)
W/SelfCheck( 5923): 	at java.lang.reflect.Method.invoke(Native Method)
W/SelfCheck( 5923): 	at java.lang.reflect.Method.invoke(Method.java:372)
W/SelfCheck( 5923): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
W/SelfCheck( 5923): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
W/SelfCheck( 5923): 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:117)
```

**Comment by mar-v-in** [comment]
Should be fixed due to commit b26d513 since v1.6.0

**Comment by wvengen** [comment]
Great :)
